atlasdb-coordination-impl/src/main/java/com/palantir/atlasdb/coordination/keyvalue/KeyValueServiceCoordinationStore.java

```diff
@@ -319,4 +332,21 @@ private static TableMetadata getCoordinationTableMetadata() {
                 false,
                 TableMetadataPersistence.LogSafety.SAFE);
     }
+
+    private class InitializingWrapper extends AsyncInitializer implements AutoDelegate_CoordinationStore<T> {
```

Comment:
The convention we used is to put this as the first item in the class

Reply:
Alright, have brought it to the front.